unit-tests/src/test/scala/scala/scalanative/unsafe/ExternTest.scala

```diff
@@ -47,14 +47,15 @@ class ExternTest {
     }
   }
 
+  @Ignore("linking error issue #1652")
   @Test def sameExternNameInTwoDifferentObjects(): Unit = {
     val bufsize = 10L
     val buf1    = stackalloc[Byte](bufsize)
     val buf2    = stackalloc[Byte](bufsize)
-    Ext1.snprintf(buf1, bufsize, c"%s", c"hello")
-    assertTrue(string.strcmp(buf1, c"hello") == 0)
-    Ext2.p(buf2, bufsize, c"%d", 1)
-    assertTrue(string.strcmp(buf2, c"1") == 0)
+    // Ext1.snprintf(buf1, bufsize, c"%s", c"hello")
+    // assertTrue(string.strcmp(buf1, c"hello") == 0)
+    // Ext2.p(buf2, bufsize, c"%d", 1)
+    // assertTrue(string.strcmp(buf2, c"1") == 0)
```

Comment:
So wait, that test worked with the old testing framework but not once ported to JUnit anymore!?

Reply:
It seems that the solution this test is testing doesn't work in some cases..
Do `@Test` methods carry some magic?

Reply:
I tried locally and I can't reproduce. The test correctly passes in my machine.

Reply:
The old test framework had the `object`s nested inside the test `object`. The JUnit is a class and nesting inside the class is not supported so I moved them to the top level. It is working locally but not in CI.

Reply:
We should find a way to preserve this test. Can you put the content of that method in a separate `def` in the companion object, and call it from this `@Test` method? Does that work around the issue?

Reply:
I put the methods in a companion object and it still fails. I'm not sure we can work around the issue. It must be related somehow to the `clang 5` in CI as the version I use on macOS and @lolgab tried on his Linux setup and it worked. Not sure what to do.

Reply:
I was able to work around the problem by putting the entire test body in the companion object.